### DISCUSSION TITLE
这次改动有点大，能否把使用文档补全

### DISCUSSION THREAD

**Discussion by Justwannasleep** [0-8-4/miui-auto-tasks#164] [综合讨论｜General]
引入的类型检查、把config文件从源码移除 但是相关的使用方法没说。特别是config.yaml文件，没有申明编码是gbk、路径是./data

**Comment by ZYMirror** [comment]
提高使用门槛，排除一些小白挺好

**Reply by 859213118** [reply]
小米社区cookie怎么抓啊，最近的格式好像和以前不一样了

**Comment by Night-stars-1** [comment]
编码是utf-8
配置文件的路径在第一次运行脚本的时候会输出位置

**Reply by Justwannasleep** [reply]
旧的配置文件直接贴进去会报错：
UnicodeDecodeError: 'gbk' codec can't decode byte 0x84 in position 327: illegal multibyte sequence
移除该配置文件重新运行得到的配置文件是U8编码，应该后奇怪字符混进去了。

**Reply by Night-stars-1** [reply]
应该是原配置文件是gbk编码的

**Comment by cy28438981** [comment]


**Reply by Night-stars-1** [reply]
啥

**Reply by cy28438981** [reply]
小米社区签到，签到的时候出滑块，造成无法签到成功，我不会配置过滑块，能请求帮忙搭建下吗

**Reply by Night-stars-1** [reply]
该项目不提供也不建议讨论过滑块

**Reply by cy28438981** [reply]
可恶的小米5级，候鸟马月才能到，不建议归不建议，所有后果我自行承担，求助大佬~

**Reply by Night-stars-1** [reply]
你无法帮别人承担法律后果，法律也不允许这样的事发生
